### PR TITLE
Use go 1.19.4 to run vulncheck

### PR DESCRIPTION
**Pull request by reivaj05**

### CODE REVIEW

**Review by dvaldivia — APPROVED** [review]

File: .github/workflows/jobs.yaml [hunk 1]
```diff
@@ -247,7 +247,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [ 1.19 ]
+        go-version: [ 1.19.4 ]
```
Comment (harshavardhana) [inline .github/workflows/jobs.yaml:250]: there is `check-latest: true` missing.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by harshavardhana** [comment]
Looks like a regression in the `govulncheck` tool

```
Scanning for dependencies with known vulnerabilities...
panic: interface conversion: types.Type is *types.Interface, not *types.Array

goroutine 1 [running]:
golang.org/x/tools/go/callgraph/vta.sliceArrayElem({0x9584c0, 0xc05f22dff0})
	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/utils.go:126 +0x1bd
golang.org/x/tools/go/callgraph/vta.(*builder).indexAddr(0xc08bfff[7](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:8)90?, 0xc0560a0600)
	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/graph.go:4[8](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:9)1 +0x38
golang.org/x/tools/go/callgraph/vta.(*builder).instr(0xc08bfff678?, {0x[9](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:10)5bc58?, 0xc0560a0600?})
	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/graph.go:355 +0x165
golang.org/x/tools/go/callgraph/vta.(*builder).fun(...)
	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/graph.go:299
golang.org/x/tools/go/callgraph/vta.(*builder).visit(0xc08bfff790, 0xc00008c400?)
	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/graph.go:291 +0x1e5
golang.org/x/tools/go/callgraph/vta.typePropGraph(...)
	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/graph.go:265
golang.org/x/tools/go/callgraph/vta.CallGraph(0xc08bfffb70?, 0xc0dc98[10](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:11)30)
	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/vta.go:75 +0xe5
golang.org/x/vuln/vulncheck.callGraph(0xc03bd6a180, {0xc0dc968000, 0x18cf, 0x100?})
	/home/runner/go/pkg/mod/golang.org/x/vuln@v0.0.0-20221208180742-f2dca5ff4cc3/vulncheck/utils.go:70 +0x2b9
golang.org/x/vuln/vulncheck.Source({0x95a450, 0xc0000360d0}, {0xc08babc000?, 0x43, 0x80}, 0xc08bb51d78)
	/home/runner/go/pkg/mod/golang.org/x/vuln@v0.0.0-20221208180742-f2dca5ff4cc3/vulncheck/source.go:80 +0x4cd
golang.org/x/vuln/internal/govulncheck.Source({0x95a450?, 0xc0000360d0?}, 0xc00014feb0?, {0xc08babc000, 0x43, 0x80})
	/home/runner/go/pkg/mod/golang.org/x/vuln@v0.0.0-20221208180742-f2dca5ff4cc3/internal/govulncheck/run.go:31 +0x94
main.doGovulncheck({0xc000024050?, 0x1, 0x1}, 0x1)
	/home/runner/go/pkg/mod/golang.org/x/vuln@v0.0.0-20221208180742-f2dca5ff4cc3/cmd/govulncheck/main.go:[11](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:12)4 +0x35b
main.main()
	/home/runner/go/pkg/mod/golang.org/x/vuln@v0.0.0-2022[12](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:13)08[18](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:19)0742-f2dca5ff4cc3/cmd/govulncheck/main.go:69 +0x1[27](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:28)
```

You can't fix this by upgrading go version j

**Comment by reivaj05** [comment]
> Looks like a regression in the `govulncheck` tool
> 
> ```
> Scanning for dependencies with known vulnerabilities...
> panic: interface conversion: types.Type is *types.Interface, not *types.Array
> 
> goroutine 1 [running]:
> golang.org/x/tools/go/callgraph/vta.sliceArrayElem({0x9584c0, 0xc05f22dff0})
> 	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/utils.go:126 +0x1bd
> golang.org/x/tools/go/callgraph/vta.(*builder).indexAddr(0xc08bfff[7](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:8)90?, 0xc0560a0600)
> 	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/graph.go:4[8](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:9)1 +0x38
> golang.org/x/tools/go/callgraph/vta.(*builder).instr(0xc08bfff678?, {0x[9](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:10)5bc58?, 0xc0560a0600?})
> 	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/graph.go:355 +0x165
> golang.org/x/tools/go/callgraph/vta.(*builder).fun(...)
> 	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/graph.go:299
> golang.org/x/tools/go/callgraph/vta.(*builder).visit(0xc08bfff790, 0xc00008c400?)
> 	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/graph.go:291 +0x1e5
> golang.org/x/tools/go/callgraph/vta.typePropGraph(...)
> 	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/graph.go:265
> golang.org/x/tools/go/callgraph/vta.CallGraph(0xc08bfffb70?, 0xc0dc98[10](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:11)30)
> 	/home/runner/go/pkg/mod/golang.org/x/tools@v0.4.0/go/callgraph/vta/vta.go:75 +0xe5
> golang.org/x/vuln/vulncheck.callGraph(0xc03bd6a180, {0xc0dc968000, 0x18cf, 0x100?})
> 	/home/runner/go/pkg/mod/golang.org/x/vuln@v0.0.0-20221208180742-f2dca5ff4cc3/vulncheck/utils.go:70 +0x2b9
> golang.org/x/vuln/vulncheck.Source({0x95a450, 0xc0000360d0}, {0xc08babc000?, 0x43, 0x80}, 0xc08bb51d78)
> 	/home/runner/go/pkg/mod/golang.org/x/vuln@v0.0.0-20221208180742-f2dca5ff4cc3/vulncheck/source.go:80 +0x4cd
> golang.org/x/vuln/internal/govulncheck.Source({0x95a450?, 0xc0000360d0?}, 0xc00014feb0?, {0xc08babc000, 0x43, 0x80})
> 	/home/runner/go/pkg/mod/golang.org/x/vuln@v0.0.0-20221208180742-f2dca5ff4cc3/internal/govulncheck/run.go:31 +0x94
> main.doGovulncheck({0xc000024050?, 0x1, 0x1}, 0x1)
> 	/home/runner/go/pkg/mod/golang.org/x/vuln@v0.0.0-20221208180742-f2dca5ff4cc3/cmd/govulncheck/main.go:[11](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:12)4 +0x35b
> main.main()
> 	/home/runner/go/pkg/mod/golang.org/x/vuln@v0.0.0-2022[12](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:13)08[18](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:19)0742-f2dca5ff4cc3/cmd/govulncheck/main.go:69 +0x1[27](https://github.com/minio/console/actions/runs/3652898175/jobs/6171784394#step:5:28)
> ```
> 
> You can't fix this by upgrading go version j

@harshavardhana i found an issue in github saying that this didn't happen with go1.19.4 so i thought in trying

**Comment by reivaj05** [comment]
Didn't work

**Pull request closed without merging**